en/04_Changelogs/5.0.0.md

```diff
@@ -447,58 +456,56 @@ Note that as part of this change the namespaces of several classes changed. Refe
 
 ### Other new features {#other-features}
 
 - [`DataObject::get_one()`](api:SilverStripe\ORM\DataObject::get_one()) can now be called directly from subclasses of `DataObject` without passing in a class as the first argument (e.g. `SiteTree::get_one(filter: ['Title:startsWith' => 'About'])`).
-- Strings returned from the [getSummary()](api:DNADesign\Elemental\Models\BaseElement::getSummary()) method in elemental blocks will automatically be displayed as the summary for the element in the [ElementalArea](api:DNADesign\Elemental\Models\ElementalArea) - this no longer needs to be manually added in [provideBlockSchema()](api:DNADesign\Elemental\Models\BaseElement::provideBlockSchema()).
+- Strings returned from the [getSummary()](api:DNADesign\Elemental\Models\BaseElement::getSummary()) method in elemental blocks are automatically displayed as the summary for the element in the [ElementalArea](api:DNADesign\Elemental\Models\ElementalArea) - this no longer needs to be manually added in [provideBlockSchema()](api:DNADesign\Elemental\Models\BaseElement::provideBlockSchema()).
 - The elemental [TopPage DataExtension](api:DNADesign\Elemental\TopPage\DataExtension) is applied to [`BaseElement`](api:DNADesign\Elemental\Models\BaseElement) and [`ElementalArea`](api:DNADesign\Elemental\Models\ElementalArea) by default, and the corresponding [`SiteTreeExtension`](api:DNADesign\Elemental\TopPage\SiteTreeExtension) is applied to [`SiteTree`](api:SilverStripe\CMS\Model\SiteTree) by default. This allows you to use `$TopPage` in your elemental block templates as a more performant alternative to `$Page` when referring to the page the element belongs to.
 - Void HTML elements such as `<img>` are no longer rendered as self-closing tags e.g. `<img src="foo" alt="bar">` is now rendered, previously `<img src="foo" alt="bar" />` was rendered.
-- It’s now possible to hook into FormField validation via the `updateValidationResult` extension hook. Module authors releasing form fields should opt-in to this functionality by calling the new `extendValidationResult` method to enable developers to extend and customise validation. See the [form validation documentation](/developer_guides/forms/validation/#extensions) for more information.
-
-## Bugfixes {#bugfixes}
-
-- If a page which is a child of a root-level page gets archived, and then its former parent is removed, it can only be restored if `can_be_root` for that page's class is true.
-
-This release includes a number of bug fixes to improve a broad range of areas. Check the change logs for full details of these fixes split by module. Thank you to the community members that helped contribute these fixes as part of the release!
```

Comment:
This seems like a really random comment to have here. If we want to keep this here, then we should move it to a more sensible place.

Reply:
This is a breaking behavioural change so it should be here somewhere. Please suggest a more sensible place.

Reply:
I think its meant to be an introduction to our full commit list. But there's an API change list in between that an the full list.

I'll move the language down there.

Reply:
Done